### PR TITLE
Add(route): add Nikkei Asia

### PR DESCRIPTION
**Pull request by rainrdx**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/nikkei-asia
```

## 新RSS检查列表 / New RSS Script Checklist
  
- [X] New Route
- [X] Documentation
  - [X] CN
  - [X] EN
- [X] 全文获取 fulltext
  - [X] Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [X] 日期和时间 date and time
  - [X] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added 
- [ ] `Puppeteer`

## 说明 / Note
Full text rss for Nikkei Asia (English)

### CODE REVIEW

**Comment by rainrdx** [comment]
@TonyRL 
Thanks for the review. I have removed the unnecessary changes in the markdown documentation file and added H3 headers. I do have retained my changes regarding the radar file because asia.nikkei.com and nikkei.com are actually two different publications (by the same owner). Please review again. Thank you.

**Comment by TonyRL** [comment]
You have to use `nikkei.com` as domain and `asia` as subdomain in radar rules.
If you use `asia.nikkei.com` as domain and `.` as subdoamin, no prompts will show up when user visit `asia.nikkei.com` with RSSHub-Radar plugin.

**Comment by rainrdx** [comment]
> You have to use `nikkei.com` as domain and `asia` as subdomain in radar rules. If you use `asia.nikkei.com` as domain and `.` as subdoamin, no prompts will show up when user visit `asia.nikkei.com` with RSSHub-Radar plugin.

@TonyRL My misunderstanding. Sorry and fixed!

File: docs/en/traditional-media.md [hunk 1]
```diff
@@ -207,6 +207,12 @@ Only `s00017` is in English.
 
 <RouteEn author="Andiedie" example="/nhk/news_web_easy" path="/nhk/news_web_easy"/>
 
+## Nikkei Asia
+
+### Latest News
+
+<Route author="rainrdx" example="/nikkei-asia" path="/nikkei-asia"/>
```
Comment (TonyRL) [inline docs/en/traditional-media.md:214]:
```suggestion
<RouteEn author="rainrdx" example="/nikkei-asia" path="/nikkei-asia"/>
```

File: lib/v2/nikkei-asia/radar.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'nikkei.com': {
+        _name: 'Nikkei Asia',
+        'asia': [
+            {
+                title: 'Nikkei Asia',
```
Comment (TonyRL) [inline lib/v2/nikkei-asia/radar.js:6]: The title here should match your H3 header in the docs

**Review by TonyRL — CHANGES_REQUESTED** [review]

File: lib/v2/nikkei-asia/index.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+
+module.exports = async (ctx) => {
+
+    const currentUrl = `https://main-asianreview-nikkei.content.pugpig.com/editionfeed/4519/pugpig_atom_contents.json`;
```
Comment (TonyRL) [inline lib/v2/nikkei-asia/index.js:8]:
```suggestion
    const currentUrl = 'https://main-asianreview-nikkei.content.pugpig.com/editionfeed/4519/pugpig_atom_contents.json';
```

File: lib/v2/nikkei-asia/index.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+		const fulltextcontent = cheerio.load(fulltext.data);
+		item.description = fulltextcontent('section[class="pp-article__body"]').html().replace(/\.\.\/\.\.\/\.\.\/\.\.\//g, 'https://main-asianreview-nikkei.content.pugpig.com/');
+                return item;
+            })
+        )
+    );
+
+    ctx.state.data = {
+        title: `Nikkei Asia`,
+        link: `https://asia.nikkei.com`,
```
Comment (TonyRL) [inline lib/v2/nikkei-asia/index.js:38]:
```suggestion
        title: 'Nikkei Asia',
        link: 'https://asia.nikkei.com',
```

**Review by TonyRL — CHANGES_REQUESTED** [review]

**Comment by rainrdx** [comment]
@TonyRL Thanks again and again for the review. All requested changes accepted.

**Pull request merged**